wgengine/router/router_linux.go

```diff
@@ -1102,6 +1106,17 @@ var ipRules = []netlink.Rule{
 	// usual rules (pref 32766 and 32767, ie. main and default).
 }
 
+var addedUDMProIPRules = false
+
+// lazy load UDM Pro rules
+func ipRules() []netlink.Rule {
```

Comment:
I renamed and wrapped `_ipRules` variable in a function so we can lazy load the rules and we can guarantee that regardless of when ipRules() is called, the rules will be added before being consumed.

Reply:
@sailorfrag I added a properly formatted docstring